cellar_rebalancer/src/allocation.rs

```diff
@@ -0,0 +1,157 @@
+use crate::{cellars::uniswapv3::UniswapV3CellarTickInfo, prelude::*, somm_send};
+use abscissa_core::Application;
+use deep_space::{Coin, Contact};
+use ethers::prelude::U256;
+use proto::query_client::QueryClient as AllocationQueryClient;
+use somm_proto::somm as proto;
```

Comment:
This layer of indirection is a bit confusing, and we also have other proto definitions in the project.